### DISCUSSION TITLE
How can I retrieve the name of user giving his ID ?

### DISCUSSION THREAD

**Discussion by mandrecavalcanti** [salman0ansari/whatsapp-api-nodejs#702] [Q&A]
How can I retrieve the name of user giving his ID ?
Using the "Get User Status", it shows only the status text
Using the "Download Profile Picture", it shows only the photo
Is there a function that retrieve the "Name" displayed below the photo ?

**Comment by salman0ansari** [comment]
I don't think there is a function for it in baileys.
